sonar-php-plugin/build.gradle.kts

```diff
@@ -0,0 +1,142 @@
+import org.sonarsource.php.enforceJarSize
+
+plugins {
+  id("org.sonarsource.php.java-conventions")
+  id("org.sonarsource.php.artifactory-configuration")
+  id("org.sonarsource.php.code-style-convention")
+  jacoco
```

Comment:
jacoco should be configured in `org.sonarsource.php.java-conventions`, not needed here